.github/workflows/docker-publish.yml

```diff
@@ -23,13 +23,13 @@ jobs:
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v1
 
-      - name: Cache Docker layers
```

Comment:
is this intentional to remove cache?

Reply:
Yes, the cache is only caching the exported layers in the final build stage.  Which provides no real benefit. This could be changed to mode=max to export all interminate layers but GitHub has a limit of 10GB across a single repo.   Also the time to prossess approximately 20gb of cache and compress it really outweighs the benefits.  A full cache will also not prossess RUN commands in the dockerfile.

Reply:
Also this cache is being accessed during pull requests.  It can create conflicts and slow the processes down.